election_reminder.py

```diff
@@ -15,7 +15,7 @@
 contests = [ ]
 
 with open('data.json') as f:
-	contests = json.load(f)
+	contests = sorted(json.load(f), key=lambda contest: dateutil.parser.parse(contest['date']))
```

Comment:
This doesn't handle the case where `date` is an invalid format or not present. Maybe a `.get` with a default.

Reply:
Done, thanks!